webos-qml/project.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "music",
+    "id": "music",
+    "projectType": "webos-qml",
+    "sourceLocale": "en-KR",
+    "pseudoLocale": ["zxx-XX", "zxx-Cyrl-XX", "zxx-Hans-XX", "zxx-Hebr-XX"],
```

Comment:
All 4 pseudo locales output to the same file: `resources/music_zxx.ts`

The output file name should be `music_zxx-[script].ts` when there is a conflict

That's a problem with the qml plugin, not this sample, so I'll approve this sample. However, the plugin should be fixed independently. (Another bug that this sample project has revealed!)

Reply:
yes, In order to get the output properly, the qml plugin should be updated as the new version. (It needs https://github.com/iLib-js/ilib-loctool-webos-qml/pull/15) I'll update the plugin version.

Reply:
Now it generates properly.